clients/src/main/resources/common/message/FetchRequest.json

```diff
@@ -73,6 +75,9 @@
           "about": "The current leader epoch of the partition." },
         { "name": "FetchOffset", "type": "int64", "versions": "0+",
           "about": "The message offset." },
+        // TODO: Are we sure we want this as a tagged field?
```

Comment:
This was mostly a note for myself to give it some thought. Long term, I think we're going to want both consumers and followers to use this field instead of relying on `OffsetsForLeaderEpoch` for validation. However, in the short term, it is likely to be unused. So the benefit of making it a tagged field is unclear. Anyway, I'll remove this since we're probably going to leave it as a tagged field for now.